spec/lib/mcb_spec.rb

```diff
@@ -94,11 +95,11 @@
         )
 
         decoded_token = Token::DecodeService.call(encoded_token: encoded_token,
-          secret: secret,
-          algorithm: encoding,
-          audience: audience,
-          issuer: issuer,
-          subject: subject)
+                                                  secret: secret,
```

Comment:
🙈

Reply:
Ooh crikey. I'll sort that out. Good spot.

Reply:
Looks like a subsequent commit fixes that.